client/app/about/about.html

```diff
@@ -12,7 +12,9 @@ <h1 class="section__title">
                     {{ 'This approach allows the distribution of tasks to many individual mappers in the context of emergency or other humanitarian mapping scenario. It also allows monitoring of the overall project progress and helps improve the consistency of the mapping (e.g., elements to cover, specific tags to use, etc.).' | translate }}
                 </p>
                 <p>
-                    All of this work is done through <a href="https://www.openstreetmap.org/" target="_top">OpenStreetMap</a>. OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit <a href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">OpenStreetMap Foundation</a>. Read more on the <a href="https://wiki.openstreetmap.org/wiki/Main_Page">OSM Wiki</a> or join the discussion with your <a href="http://wiki.openstreetmap.org/wiki/Mailing_lists#Country-Specific_Lists">local OSM community</a>.
+                    {{'All of this work is done through'| translate }} <a href="https://www.openstreetmap.org/" target="_top">OpenStreetMap</a>. {{'OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit '| translate }}<a
+                        href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">{{'OpenStreetMap Foundation'| translate }}</a>. {{'Read more on the'| translate }} <a href="https://wiki.openstreetmap.org/wiki/Main_Page">{{'OSM Wiki'| translate }}</a>{{' or join the discussion with your '| translate }}<a
```

Comment:
```suggestion
                        href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">{{'OpenStreetMap Foundation'| translate }}</a>. {{'Read more on the'| translate }} <a href="https://wiki.openstreetmap.org/wiki/Main_Page"> {{'OSM Wiki'| translate }}</a>{{' or join the discussion with your '| translate }}<a
```

Reply:
@smit1678 unfortunately due to our branch protection policy this method of applying changes will not work. I will pull these together into a PR foryou or @ethan-nelson to review and merge into `develop` (automatically putting that into this PR)